ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 import {
   ENVIRONMENT_TYPE_POPUP,
   ENVIRONMENT_TYPE_NOTIFICATION,
 } from '../../../../../shared/constants/app';
 import { getEnvironmentType } from '../../../../../app/scripts/lib/util';
-import NetworkDisplay from '../../network-display';
 import Identicon from '../../../ui/identicon';
 import { shortenAddress } from '../../../../helpers/utils/util';
 import AccountMismatchWarning from '../../../ui/account-mismatch-warning/account-mismatch-warning.component';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
-import { Icon, IconName } from '../../../component-library';
+import { Icon, IconName, PickerNetwork } from '../../../component-library';
+import { BackgroundColor } from '../../../../helpers/constants/design-system';
+import { getTestNetworkBackgroundColor } from '../../../../selectors';
```

Comment:
Let's import the `getCurrentNetwork` selector here so we can get the network logo image if available
```suggestion
import {
  getTestNetworkBackgroundColor,
  getCurrentNetwork,
} from '../../../../selectors';
```